src/System.Private.ServiceModel/tools/setupfiles/BridgeCertificateBootstrapper.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+show_usage() 
+{
+    echo ""
+    echo "WCF Core Bridge CA Certificate Bootstrapper" 
+    echo "    Usage: $0 [bridge-host] [certificate-file]"
+    echo ""
+    echo "    bridge-host - hostname where Bridge is running" 
+    echo "    certificate-file - Full file path where the retrieved certificate should be saved"
+    echo ""
+}
+
+run_bootstrapper() 
+{
+    __bridge_host=$1
+    __outfile=$2
+
+     # Locate any CoreRun.exe to be able to execute BridgeCertificateBootstrapper.exe
+     # Because BridgeCertificateBootstrapper itself is a test project, we are guaranteed
+     # it will contain CoreRun.exe even if no other tests have been built.
+    __scriptpath=$(cd "$(dirname "$0")"; pwd -P)
+    __binpath=$__scriptpath/../../../../bin
+    __testspath=$__binpath/tests
+    __corerun_path=$(dirname "$(find $__testspath -iname 'ScenarioTests.Common.dll' | head -1)")
+
+    if [ $? -ne 0 ]; then
+        echo "Could not find 'ScenarioTests.Common.dll' under path '$__binpath'" 
+        exit 1
+    fi
+
+    __corerun_exe="$__corerun_path/corerun"
+       
+    if [ ! -e "$__corerun_exe" ]; then 
+        echo "Could not find corerun in path $__corerun_path"
+        exit 1
+    fi
+
+    __bootstrapper_path=$(find $__binpath -iname 'BridgeCertificateBootstrapper.exe' | head -1)
+   
+    if [ "$__bootstrapper_path" == "" ]; then 
+        echo "Could not find 'BridgeCertificateBootstrapper.exe' under path '$__binpath'" 
+        exit 1
+    fi 
+
+    echo "Found bootstrapper at '$__bootstrapper_path'" 
+    
+    export BridgeHost=$__bridge_host
+
```

Comment:
Right now, I mandate it to be set at the script (so you'd run the script with something like `sudo ./BridgeCertificateBootstrapper.sh $BridgeHost ~/tmp/wcf-ca.crt`). The script will not clobber BridgeHost as a new shell is spun up, kind of like `setlocal` in DOS.

I _think_ it will simplify things when we bake this into the build scripts, but I'll make a call on making it optional or not as I bake things in.